docs/source/deployment/now.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Deploying with Zeit Now
+sidebar_title: Now
+description: Deploying your GraphQL server to Zeit Now
+---
+
+`now` is a service by Zeit that allows you to deploy an instance of Apollo Server and have a functional GraphQL endpoint. `now` supports three types of deployments for web apps: `static`, `node.js`, and `Docker`. We'll focus on the Node.js deployment for this guide.
+
+## Node.js Deployment
+
+Deployment to `now` for node apps simply requires a `package.json` file to be present in your app directory. Your server directory has a `package.json` file.
+
+```js
+{
+  "name": "graphqlservice",
+  "version": "1.0.0",
```

Comment:
Nothing from here down to `dependencies` is required, so let's shorten it for brevity.